Frontend/RecRock-Calendar-Frontend/src/app/calendar/components/edit-day-popup/popup-dto.ts

```diff
@@ -0,0 +1,16 @@
+export class PopupDTO {
+  constructor(
+    public name: string,
+    public startTime: string,
+    public endTime: string,
+    public date: Date,
+    public eventExists: boolean,
+    public noModification: boolean,
+    public saveEvent: boolean,
+    public deleteEvent: boolean
+  ) {}
+
+  public static formatTime(time: Date): string {
```

Comment:
I would create a folder called "helpers" in the shared module and I would add a date-time.helper.ts file. If you add this method there, it will be much easier in the future to reuse it.